### PR TITLE
[ja] Translate reference/scheduling/config/ & reference/glossary/replica-set.md into Japanese

### LINKED ISSUE #32068: [ja] Translate reference/scheduling/config/ into Japanese
This is a Feature Request

What would you like to be added

- Create new directory `content/ja/docs/reference/scheduling` and `_index.md`.
- Translate [reference/scheduling/config/](https://kubernetes.io/docs/reference/scheduling/config/) into Japanese.

Why is this needed
This page has not been translated.

Comments

/language ja
/assign

### LINKED ISSUE #32081: [ja] Translate reference/glossary/replica-set.md into Japanese
**This is a Feature Request**

**What would you like to be added**
Translate [reference/glossary/replica-set.md](https://kubernetes.io/docs/reference/glossary/?fundamental=true#term-replica-set) into Japanese.

**Why is this needed**
This page has not been translated.
It is needed for PR #32068

Comments

/language ja
/assign

### PR DESCRIPTION
**Pull request by 196Ikuchil**
Resolves #32081, resolve #32068

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @nasa9084

File: content/ja/docs/reference/scheduling/config.md [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+<!-- overview -->
+
+<!-- body -->
+
+スケジューリングプロファイルは、{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}でスケジューリングの
+異なるステージを設定することができます。
+各ステージは、拡張点に公開されています。プラグインをそれらの拡張点に1つ以上実装することで、スケジューリングの振る舞いを変更できます。
+
+KubeSchedulerConfiguration([`v1beta2`](/docs/reference/config-api/kube-scheduler-config.v1beta2/)
+か[`v1beta3`](/docs/reference/config-api/kube-scheduler-config.v1beta3/))構造体を使用して、`kube-scheduler --config <filename>`を実行することで、スケジューリングプロファイルを指定することができます。
```
Comment (nasa9084) [inline content/ja/docs/reference/scheduling/config.md:21]:
```suggestion
KubeSchedulerConfiguration([`v1beta2`](/docs/reference/config-api/kube-scheduler-config.v1beta2/)か[`v1beta3`](/docs/reference/config-api/kube-scheduler-config.v1beta3/))構造体を使用して、`kube-scheduler --config <filename>`を実行することで、スケジューリングプロファイルを指定することができます。
```

File: content/ja/docs/reference/scheduling/config.md [hunk 2]
```diff
@@ -0,0 +1,417 @@
...
+apiVersion: kubescheduler.config.k8s.io/v1beta2
+kind: KubeSchedulerConfiguration
+clientConnection:
+  kubeconfig: /etc/srv/kubernetes/kube-scheduler/kubeconfig
+```
+
+## プロファイル
+
+スケジューリングプロファイルは、{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}でスケジューリングの
+異なるステージを設定することができます。
```
Comment (nasa9084) [inline content/ja/docs/reference/scheduling/config.md:35]:
```suggestion
スケジューリングプロファイルは、{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}}でスケジューリングの異なるステージを設定することができます。
```

File: content/ja/docs/reference/scheduling/config.md [hunk 3]
```diff
@@ -0,0 +1,417 @@
...
+[プラグイン](#scheduling-plugins)をそれらの拡張点に1つ以上実装することで、スケジューリングの振る舞いを変更できます。
+
+単一の`kube-scheduler`インスタンスで[複数のプロファイル](#multiple-profiles)を実行するように設定することも可能です。
+
+### 拡張点 {#extension-points}
+
+スケジューリングは一連のステージで行われ、以下の拡張点に公開されています。
+
+1. `queueSort`: これらのプラグインは、スケジューリングキューにある`pending`状態のPodを
+   ソートするための順序付け関数を提供します。同時に有効化できるプラグインは1つだけです。
```
Comment (nasa9084) [inline content/ja/docs/reference/scheduling/config.md:46]:
```suggestion
1. `queueSort`: これらのプラグインは、スケジューリングキューにある`pending`状態のPodをソートするための順序付け関数を提供します。同時に有効化できるプラグインは1つだけです。
```

Comment (nasa9084) [reply content/ja/docs/reference/scheduling/config.md:46]: 以下同様に、文の途中での改行は消してもらえればと思います(改行するとスペースが入ってしまうはず・・・日本語は英語と違ってスペースが入らない言語なので・・・)

File: content/ja/docs/reference/scheduling/config.md [hunk 4]
```diff
@@ -0,0 +1,417 @@
...
+1. `permit`: これらのプラグインではPodのバインディングを拒む、または遅延させることができます。
+1. `preBind`: これらのプラグインは、Podがバインドされる前に必要な処理を実行できます。
+1. `bind`: これらのプラグインはPodをNodeにバインドします。`bind`プラグインは順番に呼び出され、
+   1つのプラグインがバインドを完了すると、残りのプラグインはスキップされます。`bind`プラグインは
+   少なくとも1つは必要です。
+1. `postBind`: これは、Podがバインドされた後に呼び出される情報提供のための拡張点です。
+1. `multiPoint`: このフィールドは設定のみ可能で、プラグインが適用されるすべての拡張点に対して
+   同時に有効化または無効化することができます。
+
+次の例のように、それぞれの拡張点に対して、特定の[デフォルトプラグイン](#scheduling-plugins)を無効化、または自作のプラグイン有効化することができます。
```
Comment (t-inu) [inline content/ja/docs/reference/scheduling/config.md:70]:
```suggestion
次の例のように、それぞれの拡張点に対して、特定の[デフォルトプラグイン](#scheduling-plugins)を無効化、または自作のプラグインを有効化することができます。
```

File: content/ja/docs/reference/scheduling/config.md [hunk 5]
```diff
@@ -0,0 +1,417 @@
...
+  に属するPodのNode間の拡散を優先します。
+  拡張点:`preScore`、`score`
+- `CinderLimits`:Nodeが[`OpenStack Cinder`](https://docs.openstack.org/cinder/)
+  ボリューム制限を満たせるかチェックします。
+  拡張点:`filter`
+
+### 複数のプロファイル {#multiple-profiles}
+
+`kube-scheduler`は複数のプロファイルを実行するように設定することができます。
+各プラグインは関連するー名を持ち、その[拡張点](#extension-points)に異なるプラグインを
```
Comment (t-inu) [inline content/ja/docs/reference/scheduling/config.md:159]:
```suggestion
各プロファイルは関連するスケジューラー名を持ち、その[拡張点](#extension-points)に異なるプラグインを
```
> Each profile has an associated scheduler name and can have a different set of plugins configured in its extension point.

File: content/ja/docs/reference/scheduling/config.md [hunk 6]
```diff
@@ -0,0 +1,417 @@
...
+        - name: '*'
+      score:
+        disabled:
+        - name: '*'
+```
+
+`v1beta3`では、`MultiPoint`を通じて、内部的に全ての[デフォルトプラグイン](#scheduling-plugins)が有効化されています。
+しかしながら、デフォルト値(並び順やスコアの重みなど)を柔軟に設定し直せるように、個別の拡張点は用意されています。
+例えば、2つのスコアプラグイン`DefaultScore1`と`DefaultScore2`に、重み1が設定されているとします。
+その場合、次のようにに重さを変更し、並べ替えることができます
```
Comment (t-inu) [inline content/ja/docs/reference/scheduling/config.md:267]:
```suggestion
その場合、次のように重さを変更し、並べ替えることができます
```

**Review by t-inu — COMMENTED** [review]
I left comments because I found some typos and so on.

**Pull request merged**